tests/tests/control_csawmgt

```diff
@@ -0,0 +1,40 @@
+###############################################################################
+#
+#  Global control csawmgt test
```

Comment:
I am a little confused what the "control" stands for in all these tests. I thought control is GFS v16, and any suffix describes deviations from GFS v16 control (e.g. _flake, _2threads). But csawmg is an entirely different suite, for example.

Reply:
Yes, the control test is GFS_v16. Any suffix test has new updates from GFS_v16, it could be additional new features or new physics packages replacing current one used in opn GFS v16. In the case of csawmgt, the setting is based in L127 GFS_v16 test, only the related physics namelist are changed, the same for control_noahmp, control_rrtmgp, control_thompson, etc.